Create CONTRIBUTING.md (#13)
openid/federation: CONTRIBUTING.md

# How to contribute

Contributions to the OpenID Federation specification are welcome. Please ensure that your contributions adhere to the following guidelines:

- **Issues**: Use the issue tracker to report problems or suggest enhancements.

> - **Pull Requests**: Submit pull requests linked to Issues that was approved during the Working Group call.

Reviewer: Small grammar suggestion:

```suggestion
- **Pull Requests**: Submit pull requests linked to Issues that were approved during a Working Group call.
```
